src/components/ExpandableCard.tsx

```diff
@@ -107,111 +36,121 @@ const ExpandableCard: React.FC<IProps> = ({
-      display: "none",
-    },
-    expanded: {
-      display: "inline-block",
-    },
-  }
-
-  const fadeInOut = {
-    collapsed: {
-      opacity: 0,
-      transition: {
-        duration: 0.1,
-      },
-    },
-    expanded: {
-      opacity: 1,
-      transition: {
-        duration: 0.4,
-      },
-    },
-  }
   const matomo = {
     eventAction: `Clicked`,
     eventCategory: `ExpandableCard${eventCategory}`,
     eventName,
   }
+  const onClick = () => {
+    // Card will not collapse if clicking on a link or selecting text
+    if (
+      window.getSelection()?.toString().length === 0 &&
+      !(window.event?.target as HTMLDivElement)?.className.includes(
+        "ExternalLink"
+      )
+    ) {
+      !isVisible && trackCustomEvent(matomo)
+      setIsVisible(!isVisible)
+    }
+  }
+
   return (
-    <Card
-      onClick={() => {
-        // Card will not collapse if clicking on a link or selecting text
-        if (
-          window.getSelection()?.toString().length === 0 &&
-          !(window.event?.target as HTMLDivElement)?.className.includes(
-            "ExternalLink"
-          )
-        ) {
-          !isVisible && trackCustomEvent(matomo)
-          setIsVisible(!isVisible)
-        }
+    <Accordion
+      border="1px solid"
+      borderColor="border"
+      borderRadius="2px"
+      display="flex"
+      flex-direction="column"
+      marginBottom="1rem"
+      cursor="pointer"
+      _hover={{
+        backgroundColor: "ednBackground",
       }}
+      borderBottomWidth="0"
+      index={isVisible ? [0] : []}
     >
-      <Content>
-        <Question>
-          <Header>
-            {!!Svg && <Svg />}
-            <Title>{title}</Title>
-          </Header>
-          <TextPreview>{contentPreview}</TextPreview>
-        </Question>
-        <ButtonContainer
-          onClick={() => {
-            trackCustomEvent(matomo)
-            setIsVisible(!isVisible)
-          }}
-        >
-          {!isVisible && (
-            <ButtonLink onClick={() => setIsVisible(true)}>
-              <Translation id="more" />
-            </ButtonLink>
-          )}
-          {isVisible && (
-            <ButtonLink onClick={() => setIsVisible(false)}>
-              <Translation id="less" />
-            </ButtonLink>
-          )}
-        </ButtonContainer>
-      </Content>
-      <motion.div
-        variants={expandCollapse}
-        animate={isVisible ? "expanded" : "collapsed"}
-        initial={false}
-      >
-        <motion.div
-          variants={showHide}
-          animate={isVisible ? "expanded" : "collapsed"}
-          initial={false}
-        >
-          <Text
-            variants={fadeInOut}
-            animate={isVisible ? "expanded" : "collapsed"}
-            initial={false}
+      <AccordionItem borderTopWidth="0" flex="1">
+        <h3 style={{ margin: 0 }}>
+          <AccordionButton
+            width="100%"
+            padding={`1.5rem 1.5rem ${isVisible ? "0" : "1.5rem"}`}
```

Comment:
Lets keep this padding for the entire box and remove the extra bottom margin from the title section (see the following comment)
```suggestion
            p={6}
```